client/src/partials/group_invoice/group_invoice.js

```diff
@@ -1,193 +1,217 @@
 angular.module('bhima.controllers')
-.controller('groupInvoice', [
-  '$scope',
-  '$translate',
-  'connect',
-  'validate',
-  'messenger',
-  'uuid',
-  'SessionService',
-  function ($scope, $translate, connect, validate, messenger, uuid, SessionService) {
-
-    var dependencies = {};
-    $scope.action = '';
-    $scope.convention = '';
-    $scope.selected = {};
-    $scope.paying = [];
-
-    dependencies.invoices = {
-      query : 'ledgers/debitor/'
-    };
-
-    dependencies.conventions = {
-      required: true,
-      query : {
-        tables : {
-          'debitor_group'  : {
-            columns : ['uuid', 'name', 'account_id']
-          }
-        },
-        where : ['debitor_group.is_convention<>0', 'AND']
-      }
-    };
-
-    dependencies.debitors = {
-      required : true,
-      query : {
-        tables : {
-          'debitor' : {
-            columns : ['uuid', 'text']
-          },
-          'debitor_group' : {
-            columns : ['account_id']
-          }
+.controller('GroupInvoiceController', GroupInvoiceController);
+
+
+GroupInvoiceController.$inject = [
+  '$scope', '$translate', 'connect', 'validate', 'messenger', 'uuid', 'SessionService',
+];
+
+function GroupInvoiceController($scope, $translate, connect, validate, messenger, uuid, SessionService) {
+  var vm = this;
+
+  var dependencies = {};
+  $scope.action = '';
```

Comment:
You still using ```$scope```, every information to expose must be attached to the ```vm```.
You don't need $scope here.